### PR TITLE
[provisioning] Uninstalling cmake errors are 'fine', right?

### PR DESCRIPTION
**Pull request by chamons**

### CODE REVIEW

File: jenkins/system-dependencies.sh [hunk 1]
```diff
@@ -612,7 +612,8 @@ function install_cmake ()
 		git checkout "$BREW_CMAKE_HASH"
 		# Uninstall any existing cmakes. Ignore failures (which may happen if new cmake is installed)
 		if type -t cmake > /dev/null; then
-			brew uninstall --force cmake
+			brew uninstall --force cmake || true
```
Comment (stephen-hawley) [inline jenkins/system-dependencies.sh]: That's a thing alright.

**Comment by chamons** [comment]
Looks like we are using https://github.com/xamarin/binding-tools-for-swift/pull/326

**Comment by chamons** [comment]
:man_facepalming:

**Comment by rolfbjarne** [comment]
This seems to be working!

**Pull request merged**